data/campaigns/Eastern_Invasion/scenarios/02_The_Escape_Tunnel.cfg

```diff
@@ -1,530 +1,625 @@
-        [/scroll_to]
-        [remove_shroud]
-            side=1
-            x,y=22,5
-            radius=1
-        [/remove_shroud]
-        [delay]
-            time=750
-        [/delay]
-
-        [scroll_to]
-            x,y=28,6
-        [/scroll_to]
-        [remove_shroud]
-            side=1
-            x,y=28,6
-            radius=1
-        [/remove_shroud]
-        [delay]
-            time=750
-        [/delay]
-
-        [scroll_to]
-            x,y=34,5
-        [/scroll_to]
-        [remove_shroud]
-            side=1
-            x,y=34,5
-            radius=1
-        [/remove_shroud]
-        [delay]
-            time=750
-        [/delay]
-
-        [message]
-            speaker=Dacyn
-            message= _ "We must make haste to escape before the undead catch up."
-        [/message]
-
-        [message]
-            speaker=Gweddry
-            message= _ "Wait, before we go anywhere — who were those undead?"
-        [/message]
-
-        [message]
-            speaker=Dacyn
-            message= _ "I will explain later. For now, suffice to say they are much too powerful for us; our only hope is to head north."
-        [/message]
-    [/event]
-
-    [event]
-        name=moveto
-        [filter]
-            x,y=29,10
+#
+# Simple and easy, especially compared to S01.
+#
+# This is beatable without any combat or recruiting - just run straight through the dwarfish tunnels, and wade through the river.
+# More ambitious players can take the north path and raid the troll trezzur hole for some gold and a powerful item
+# REALLY ambitous players can fight their way through the troll leader for extra XP.
+#
+
+#define TURNS
+    {ON_DIFFICULTY 15 13 11}#enddef # undead arrival plus 3. I don't like tight turn limits, but couldn't think of another way to discourage farming the undead
+
+    [scenario]
+        id=02_The_Escape_Tunnel
+        name= _ "The Escape Tunnel"
+        map_file=02_The_Escape_Tunnel.map
+        turns={TURNS}
+        next_scenario=03_An_Unexpected_Appearance
+        victory_when_enemies_defeated=no
+
+        {UNDERGROUND}
+
+        [story]
+            [part]
+                story=_ "Screams and shouts echoed across the surface as Gweddry rushed his remaining men into the roughly hewn tunnel. A foul stench wafted up from below..."
+                music=silence.ogg
+                background=story/S02.jpg
+            [/part]
+        [/story]
+        {EI_TRACK {JOURNEY_02_NEW} }
+        {SCENARIO_MUSIC the_deep_path.ogg}
+
+        # wmllint: validate-off
+        [side]
             side=1
-        [/filter]
-
-        [message]
-            speaker=unit
-            image=scenery/signpost.png
-            message= _ "TROL TREZZUR HOLE: KEEP OWT" # wmllint: no spellcheck
-        [/message]
-
-        [allow_undo][/allow_undo]
-    [/event]
-
-    # Find the dwarves
-    [event]
-        name=sighted,moveto
-        [filter]
+            controller=human
+            gold=120
```

Comment:
why remove the variance here? I needed the extra gold I added previously

Reply:
I haven't varied player gold on any scenarios - mostly, I don't think increasing player gold makes low difficulties more relaxed to play. (plus fewer variables makes balance simpler)

Having more units means the player has to control more units. While that makes it easier to win, it's not necessarily an easier experience, particularly for the inexperienced players likely to play on easy difficulty in the first place.

If you feel any scenario is disproportionately difficult on one particular difficulty, I'd rather balance by lowering the amount of enemies on those difficulties.

> I needed the extra gold I added previously
Note that almost every scenario is balanced drastically differently from the original EI; I suggest against using the original as a balance benchmark.

Reply:
> I haven't varied player gold on any scenarios - mostly, I don't think increasing player gold makes low difficulties more relaxed to play.

OK, well, I do.

> Having more units means the player has to control more units.

Not necessarily, it just means you have more units to spare, and have more of a buffer in case you lose some, which happens a lot to me. I like being able to have enough gold to replace the units that I lose.

> While that makes it easier to win, it's not necessarily an easier experience, particularly for the inexperienced players likely to play on easy difficulty in the first place.

I think it is.

Reply:
To clarify, do you feel the issue is that easy/normal are too difficult, or that non-varying gold is bad scenario design?

Reply:
I think that non-varying gold is bad scenario design. In feedback across many campaign, I'm constantly seeing complaints from players on easy that easy is never easy enough, and players on hard that hard is never hard enough. Players on easy often need more help than scenario designers think they do, so it is best to give them as many additional tools as possible to ensure their success. Plus greater variance across difficulties provides greater replayability, as there is more of an incentive to try the other difficulties if I know there are going to be more things that are different in them.

Reply:
Have you analysed why the players on easy were finding it too hard? I haven't, but I'm wondering if part of the reason is that they've been spreading XP over too many units in previous scenarios, combined with having less XP available due to the previous enemy's lower gold.

Reply:
> In feedback across many campaign, I'm constantly seeing complaints from players on easy that easy is never easy enough

I definitely agree with this; I was shocked when I found one playtester was having trouble with one of an earlier version's easy-branch scenario on easy difficulty.  Since then, I tried to make easy (at least the easy-branch) scenarios so laughably simple that you do barely anything and still win.

I guess I'm ok with globally increasing easy-difficulty gold across all scenarios if you really feel it's necessary, but if any particular scenario is problematic I'd prefer to reduce the enemy strength instead.  My general guideline when designing scenarios was 1x/3x/5x enemies on easy/normal/hard.

I tried to make easy so incredibly easy that it would pain my soul if it's still too hard, but player experience always comes first.

> players on hard that hard is never hard enough. 

I always approve of making hard harder 😈

> Plus greater variance across difficulties provides greater replayability, as there is more of an incentive to try the other difficulties if I know there are going to be more things that are different in them.

Interesting, I've actually tried hard to ensure the opposite.  I don't want a player to feel like they're missing out on content because they decide to play a lower difficulty vs a higher difficulty; IMO difficulty should purely be a difficulty slider, and not affect the available content whatsoever.

I tried to achieve replayability through branching content instead, but still all available on the same difficulty.

Reply:
> I tried to achieve replayability through branching content instead, but still all available on the same difficulty.

I think this is a good approach. Nonetheless, the point about balancing may still stand (I'm not an expert in that regard).